api/proto/attest/attest.proto

```diff
@@ -6,38 +6,73 @@ syntax = "proto3";
 option go_package  = "github.com/lf-edge/eve/api/go/certs";
 option java_package = "com.zededa.cloud.uservice.proto";
 
-// ZAttestNonceResp is response payload for POST /api/v1/edgeDevice/attestNonce
-// or /api/v2/edgeDevice/attestNonce. It is an empty POST from the client
-// The message is assumed to be protected by a TLS session bound to the
-// device certificate for v1.
-// The message is assumed to be protected by signing envelope for v2.
+// This is the request payload for POST /api/v2/edgeDevice/id/<uuid>/attest
+// The message is assumed to be protected by signing envelope
+message ZAttestReq {
+    ZAttestReqType reqType = 1;  //sub-type of the request
+    ZAttestQuote quote = 2;      //attestation quote msg
+    repeated ZEveCert certs = 3; //X509 certs in .PEM format, signed by device certificate
```

Comment:
Didn't see the "signed" part. Is the idea that they will not be signed by themselves, but by the one and only device cert. Sounds cool!

Reply:
yes, that is the idea.